SpecialOperations2.Altis/sos/vehicle/functions/fn_activateServicePoint.sqf

```diff
@@ -19,9 +19,10 @@ private ["_vehicle", "_actionName", "_actionId"];
 _vehicle = vehicle player;
 
 //Checks for damage, fuel and touching ground (cant move people out of flying choppers, also ammo later on)
+    
 
 if (((damage _vehicle) > 0) or ((fuel _vehicle) < 1)) then {
-    _actionId = _vehicle addAction [
+    _actionId = driver _vehicle addAction [
```

Comment:
might as well be
```javascript
_actionId = player addAction [
```
because you **have** to be the driver for the service point to fire on your machine.